bin/celery-scheduler.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd /home/ali/Desktop/Assesment/app/
```

Comment:
please study about these paths.. what if the code is deployed on some other machine, then this path wont work?